mod/01_rating/set/type/metric_value.rb

```diff
@@ -24,6 +24,14 @@ def company_card
   Card.fetch company_name
 end
 
+def value_type
+  if (value_type_card = Card.fetch "#{metric_card.name}+value type") &&
```

Comment:
"value type" should have a codename, and :value_type should be a card accessor.  then instead of this fetch you'd just do `metric_card.value_type_card`.  Any time you're typing a card's name in code, it's wrong.